views/layouts/main.handlebars

```diff
@@ -29,27 +29,31 @@
           </div>
         </div>
       {{else}}
+      <div class="flex justify-between w-full">
         <a
           href="/"
           class="font-semibold hover:text-zinc-800 transition-all"
         >Home</a>
-        <a
-          href="/users/signup"
-          class="font-semibold hover:text-zinc-800 transition-all"
-        >Signup</a>
-        <a
-          href="/users/login"
-          class="font-semibold hover:text-zinc-800 transition-all"
-        >Login</a>
+        <div class="flex gap-4">
+          <a
+            href="/users/signup"
+            class="font-semibold hover:text-zinc-800 transition-all"
+          >Signup</a>
+          <a
+            href="/users/login"
+            class="font-semibold hover:text-zinc-800 transition-all"
+          >Login</a>
+        </div>
+      </div>
       {{/if}}
     </header>
-    <main class="flex flex-col min-h-[calc(100vh-80px)]">
-      <div class="flex flex-1 flex-col h-full">
+    <main class="flex grow">
+      <div class="flex flex-1 flex-col justify-center p-4">
```

Comment:
I think you could move the `flex-col justify-center p-4` from the div to the main tag and get rid of the div tag overall.

Reply:
Yup, think you're right. I'll make those changes.